precompiles/xvm/src/tests.rs

```diff
@@ -50,6 +51,7 @@ fn wrong_argument_reverts() {
                     .write(0u8)
                     .write(Bytes(b"".to_vec()))
                     .write(Bytes(b"".to_vec()))
+                    .write(U256::one())
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `write()` function is called with `U256::one()`, but it seems that the value is not being used. Please verify if this line of code is necessary and remove it if it's not.
```diff
-                     .write(U256::one())
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->